tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -452,8 +463,8 @@ public:
       auto rColl = fColls[0];
       for (unsigned int i = 1; i < fColls.size(); ++i) {
          auto &coll = fColls[i];
-         for (T &v : *coll) {
-            rColl->emplace_back(v);
+         for (const T &v : *coll) {
```

Comment:
@bluehood There was a reason not having `const` here? This fixed the range warnings in cxx17 if passing the reference to the function.